requirements.txt

```diff
@@ -1,8 +1,8 @@
 click >= 7.0
 cloudpickle >=1.3.0
 croniter >= 0.3.24, <1.0
-dask >= 2.17.0
-distributed >= 2.17.0
+dask >= 2.17.0, <2021.04.0
```

Comment:
Is it easy to parametrize this bound by python version?

Reply:
yeah, looks like there's a way to do this, trying now